types/wlr_screencopy_v1.c

```diff
@@ -21,7 +21,6 @@ struct screencopy_damage {
 	struct pixman_region32 damage;
 	struct wl_listener output_precommit;
```

Comment:
This can probably be removed?

Reply:
The precommit handler? It's what accumulates damage per output on the client object.

Reply:
Ah, sorry, not this one. But `wlr_screencopy_frame_v1.output_precommit` can be removed, now that the `wl_signal_add` call has been removed from `frame_handle_copy`.

Reply:
`wlr_screencopy_frame_v1.output_precommit` is removed now